native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -5,18 +5,24 @@ class ViewController: UIViewController {
     @IBOutlet weak var originSpinner: UIPickerView!
     @IBOutlet weak var destinationSpinner: UIPickerView!
     @IBOutlet weak var submitButton: UIButton!
+    @IBOutlet weak var departureTable: UITableView!
     
     private var stations: [String] = [String]()
+    private var departures: [DepartureInformation] = [DepartureInformation]()
     
     private let presenter: ApplicationContractPresenter = ApplicationPresenter()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         presenter.onViewTaken(view: self)
-        
+
+        departureTable.delegate = self
+        departureTable.dataSource = self
+        self.registerTableViewCells()
```

Comment:
Could we pull these three lines out into a setUpTable function we call here and declare with the UITableViewDelegate extension section